src/OpenLoco/src/GameCommands/CloneVehicle.cpp

```diff
@@ -99,22 +99,22 @@ namespace OpenLoco::Vehicles
         {
             // Do not cache this as it will be a different value every iteration
             auto chosenOffset = newHead->sizeOfOrderTable - 1;
-            GameCommands::do_35(newHead->id, order->getRaw(), chosenOffset);
+            do_35(newHead->id, order->getRaw(), chosenOffset);
         }
 
         // Copy express/local
-        if ((existingTrain.veh1->var_48 & Flags48::expressMode) != Flags48::none)
+        if ((existingTrain.veh1->var_48 & Vehicles::Flags48::expressMode) != Vehicles::Flags48::none)
         {
-            GameCommands::do12(newHead->id, 2);
+            do12(newHead->id, 2);
```

Comment:
size note, do we know what `do12` is?

Reply:
Yeah, it's currently known as `vehicleLocalExpress`. Actually, #1957 gets rid of `do12` in favour of a game command argument struct.